boards/arm/rddrone_fmuk66/rddrone_fmuk66.dts

```diff
@@ -310,6 +311,22 @@ zephyr_udc0: &usbotg {
 	status = "okay";
 	pinctrl-0 = <&i2c1_default>;
 	pinctrl-names = "default";
+
+	/*
+	 * This board does not have a BME280, it has a BMP280.
+	 * The two parts are incredibly similar, so the BME280 driver should work.
+	 */
```

Comment:
"should" or "works"? has this been tested properly?

Reply:
@gmarull .. Yes it works and has been tested thoroughly, we will edit the comment